ludwig/encoders/date_encoders.py

```diff
@@ -232,34 +253,33 @@ def __init__(
             default_weights_regularizer=weights_regularizer,
             default_bias_regularizer=bias_regularizer,
             default_activity_regularizer=activity_regularizer,
-            # default_weights_constraint=weights_constraint,
-            # default_bias_constraint=bias_constraint,
             default_norm=norm,
             default_norm_params=norm_params,
             default_activation=activation,
             default_dropout=dropout,
         )
 
-    def call(
+    def forward(
             self,
-            inputs,
-            training=None,
-            mask=None
-    ):
+            inputs: torch.Tensor,
+            training: Optional[bool] = None,
```

Comment:
You can remove the `training` and `mask` arguments here as well while calling the forward method of each encoder (e.g. `year_fc`). Pytorch doesn't need the training `flag` for backprop, and I believe we aren't using the `mask` arg anymore.

Reply:
Done.

Reply:
Should I/we also clean up, probably in a separate PR, the `training` and `mask` parameters in Encoder base class? I'm getting a warning about it.